src/main/kotlin/graphics/scenery/controls/behaviours/VRSelectionWheel.kt

```diff
@@ -48,15 +52,15 @@ class VRSelectionWheel(
      */
     override fun drag(x: Int, y: Int) {
 
-        val (closestSphere, distance) = activeWheel?.closestActionSphere() ?: return
+        val (closestSphere, distance) = activeWheel?.closestActionSphere(controller.worldPosition()) ?: return
 
         if (distance > cutoff) {
             activeWiggler?.deativate()
             activeWiggler = null
 
-        } else if (activeWiggler?.target != closestSphere.sphere.spatial()) {
+        } else if (activeWiggler?.target != closestSphere.representation.spatial()) {
             activeWiggler?.deativate()
-            activeWiggler = Wiggler(closestSphere.sphere.spatial(), 0.01f)
+            activeWiggler = Wiggler(closestSphere.representation.spatial(), 0.01f)
```

Comment:
No complaints here, just love the name `activeWiggler` ❤️ 😂

![](https://media.giphy.com/media/lNIvPaHZjPH7AP0urN/giphy.gif)